CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.65.0 / 2022-05-04
+
+The main change introduced by this release is the new v1alpha1 `ScrapeConfig` CRD.
+This implements the [proposal](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/proposals/202212-scrape-config.md)
+documented in [#5279](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/proposals/202212-scrape-config.md)
+and provides a Kubernetes native API to create and manage additional scrape configurations.
+
+* [CHANGE] Add validation for missing CRDs. #5476
```

Comment:
it should probably be a BUGFIX since the original issue was that the `PrometheusOperatorNotReady` alert would fire if the PrometheusAgent CRD wasn't deployed. At least it's not a CHANGE.